workflow/rules/ref.smk

```diff
@@ -78,9 +78,11 @@ rule tabix_known_variants:
 
 rule bwa_index:
     input:
-        "refs/genome.fasta"
+        "results/refs/genome.fasta"
     output:
-        multiext("refs/genome.fasta", ".amb", ".ann", ".bwt", ".pac", ".sa")
+        multiext("results/refs/genome", ".amb", ".ann", ".bwt", ".pac", ".sa")
```

Comment:
We actually have a divergence here. The multiext needs .fasta, and the params can be removed.

Reply:
Actually I think both is valid and supported.

Reply:
but I changed it back

Reply:
The thing is, when using a manually set prefix here, you have to ensure that it is always a prefix of the output, even if the output is changed via --default-remote-prefix (in the cloud). Hence, the code becomes more complicated, because you will need a function for the prefix. Since it is easier to just omit the prefix I voted for that.